circuit_breaker.gemspec

```diff
@@ -59,8 +59,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ["History.txt", "README.txt"]
   s.rdoc_options = ["--main", "README.txt", "--charset=UTF-8"]
 
-  s.add_runtime_dependency "aasm"
-
+  s.add_development_dependency "aasm"
```

Comment:
I prefer to only include dev deps in the gemspec when required to run tests.  I'm thinking the tests should be run with asm and without, and the 'with' can be achieved using a BUNDLE_GEMFILE and eval_gemfile, or use the 'appraisals' gem